src/Apps/Order/Routes/Shipping2/Shipping2.jest.tsx

```diff
@@ -781,13 +794,15 @@ describe("Shipping", () => {
               recommendedAddress
             )
 
-            mockCommitMutation.mockResolvedValueOnce(
-              settingOrderShipmentSuccess
-            )
+            mockCommitMutation
+              // .mockResolvedValueOnce(settingOrderShipmentSuccess)
```

Comment:
I think i commented this out while testing something else and forgot to put it back, but the tests pass. Maybe it is already mocked by default.